src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -381,7 +308,7 @@ public void startTransportService(TransportService transportService) {
             false,
             false,
             InitializeExtensionsRequest::new,
-            (request, channel, task) -> channel.sendResponse(handleExtensionInitRequest(request))
+            (request, channel, task) -> channel.sendResponse(extensionsInitRequestHandler.handleExtensionInitRequest(request, this))
```

Comment:
Why do we need to pass `this` here?

Reply:
Thank you Dan @dbwiddis to answer this question for me.